gdb_start.sh

```diff
@@ -0,0 +1,2 @@
+target remote localhost:6666
```

Comment:
don't include this file in the PR. (can use `git rm --cached` to remove it from the branch while keeping the physical file).